src/miner.cpp

```diff
@@ -74,321 +74,406 @@ int64_t UpdateTime(CBlockHeader* pblock, const Consensus::Params& consensusParam
     return nNewTime - nOldTime;
 }
 
-CBlockTemplate* CreateNewBlock(const CChainParams& chainparams, const CScript& scriptPubKeyIn,bool blockstreamCoreCompatible);
+BlockAssembler::BlockAssembler(const CChainParams &_chainparams) : chainparams(_chainparams)
```

Comment:
Importing code that doesn't initialize its members is not "cleanup" in my opinion, its very much a bug being imported.